leakcanary-sample/src/main/java/com/example/leakcanary/ExampleApplication.java

```diff
@@ -26,6 +26,7 @@
 
   @Override public void onCreate() {
     super.onCreate();
+    //noinspection ConstantConditions
```

Comment:
It's an IDE warning when toggling the `release` variant for `leakcanary-sample` since `LeakCanary.isInAnalyzerProcess()` in `leakcanary-android-no-op` evaluates to `false`.